### ISSUE TITLE
scsh documentation

### ISSUE THREAD

**Issue by reirob** [scheme/scsh#10]
Hello,

I finally made it to compile scsh. And it seems to work. Then I started to look for a documentation of scsh on scsh.net and it says "The Scsh Manual, packed with the distribution, in various formats". As well in the same page it says there is a github repo https://github.com/rmloveland/scsh-manual-texinfo.

I think it would be great to provide the manual together with the scsh at least as text file, even better as a man page.

Disclaimer: I never programmed in Scheme or Lisp an it is my first github issue request, so please correct me if I missed something.

**Comment by spelufo** [comment]
I had found that online manual a while ago. I only now found [this paper](ftp://ftp.cs.indiana.edu/pub/scheme-repository/imp/scsh-paper.ps) which makes for a much better introduction. I'd recommend linking to it in the readme.

**Comment by roderyc** [comment]
@spelufo you're right, that paper does a good job of giving color to the project. On the other hand it's from 1994 and doesn't target a modern audience. I'll try and incorporate elements of it into the new documentation.
